passes/stack.yaml

```diff
@@ -5,7 +5,7 @@ ghc-options: {'*': -fenable-rewrite-rules -threaded -funbox-strict-fields -fcons
 packages:
 - {location: .}
 - {location: ../core}
-- {location: ../stdlib}
+- {location: ../../stdlib}
```

Comment:
This is quite gross to me, but I'm not sure how to solve it. It serves the luna-manager develop directory layout, but is an eyesore. Can I get a review?